packages/graphql/tests/tck/aggregations/where/composite.test.ts

```diff
@@ -63,10 +63,10 @@ describe("Cypher Aggregations where with count and node", () => {
             CALL {
```

Comment:
This file is a new tests in translation layer, I've updated it with the queries being generated in 4.0.0 branch